zipline/data/_minute_bar_internal.pyx

```diff
@@ -112,7 +128,7 @@ def find_last_traded_position_internal(
 
     minute_pos = int_min(
         find_position_of_minute(market_opens, market_closes, end_minute,
```

Comment:
hm, true. But then you're essentially replicating `find_position_of_minute`'s logic here (the searchsorted call, plus the math).  

I think I prefer it the way it is.